user/src/main/resources/application.yml

```
@@ -55,7 +69,7 @@ management:
   endpoints:
     web:
       exposure:
-        include: ["*"]
+        include: [ "*" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)